apifuzzer/base_template.py

```diff
@@ -34,11 +35,26 @@ def __init__(self, name):
         :param cookies: (optional) Dict or CookieJar object to send with the :class:`Request`.
         """
 
+    def strategy_default(self, template):
+        for name, field in self.field_to_param.items():
+            if list(field):
+                template.append_fields([Container(name=name, fields=field)])
+
+    def strategy_all_params_at_once(self, template):
+        for name, field in self.field_to_param.items():
+            print(name)
```

Comment:
use logger instead of print.

Reply:
I think you're reviewing an old commit, this got removed.
I should squash my commits before opening a MR to make this easier for everyone.

Reply:
Yes I did :-)